firefox-ios/RustFxA/FxAWebViewModel.swift

```diff
@@ -137,21 +139,8 @@ class FxAWebViewModel: FeatureFlaggable {
                         }
                     }
                 case let .qrCode(url):
-                    accountManager.beginPairingAuthentication(
-                        pairingUrl: url,
-                        entrypoint: "pairing_\(entrypoint)",
-                        // We ask for the session scope because the web content never
-                        // got the session as the user never entered their email and
-                        // password
-                        scopes: [OAuthScope.profile, OAuthScope.oldSync, OAuthScope.session]
-                    ) { [weak self] result in
-                        guard let self = self else { return }
-
-                        if case .success(let url) = result {
-                            self.baseURL = url
-                            completion(self.makeRequest(url), .qrPairing)
-                        }
-                    }
+                    self.baseURL = url
+                    completion(self.makeRequest(url), .qrPairing)
```

Comment:
it seems a bit weird for us to call the `getManageAccountURL` again and to trigger this code regardless if it succeeds or fails here. Outside of the scope of this bug though.

Reply:
I definitely don't love that we're calling `getManageAccountURL` again. As an intermediary step, do you think it makes sense to bring the `qrCode` case outside the `getManageAccountURL` completion block?

Reply:
Lets look into this when you are back for improvements although gave you an r+ for basic functionality